internal/network/manifests_generator.go

```diff
@@ -81,13 +81,23 @@ address=/apps.{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}/{{.HOST_IP}}
 address=/api-int.{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}/{{.HOST_IP}}
 `
 
+const unmanagedResolvConf = `
+[main]
+rc-manager=unmanaged
+`
+
 const forceDnsDispatcherScript = `
 export IP="{{.HOST_IP}}"
+export BASE_RESOLV_CONF=/run/NetworkManager/resolv.conf
 if [ "$2" = "dhcp4-change" ] || [ "$2" = "dhcp6-change" ] || [ "$2" = "up" ] || [ "$2" = "connectivity-change" ]; then
     if ! grep -q "$IP" /etc/resolv.conf; then
-      sed -i "s/{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}//" /etc/resolv.conf
-      sed -i "s/search /search {{.CLUSTER_NAME}}.{{.DNS_DOMAIN}} /" /etc/resolv.conf
-      sed -i "0,/nameserver/s/nameserver/nameserver $IP\nnameserver/" /etc/resolv.conf
+      export TMP_FILE=$(mktemp /etc/forcedns_resolv.conf.XXXXXX)
+      cp  $BASE_RESOLV_CONF $TMP_FILE
+      chmod --reference=$BASE_RESOLV_CONF $TMP_FILE
+      sed -i -e "s/{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}//" \
+      -e "s/search /search {{.CLUSTER_NAME}}.{{.DNS_DOMAIN}} /" \
+      -e "0,/nameserver/s/nameserver/nameserver $IP\nnameserver/" $TMP_FILE
```

Comment:
```suggestion
      -e "s/search /&{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}} /" \
      -e "0,/nameserver/s/nameserver/& $IP\n&/" $TMP_FILE
```

No need to repeat words so much